src/Raven.Client/Documents/Changes/ChangeNotification.cs

```diff
@@ -57,9 +52,7 @@ public DynamicJsonValue ToJson()
                 [nameof(Type)] = Type.ToString(),
```

Comment:
Could we pass here just the numeric value? this way we could save some perf on the ToJson and on the FromJson?

Reply:
hmm.. no I see that it'll probably be a backward compatability break...

Reply:
And given the choice between have `Type=3` and `Type='Document'`, I ll take the readable option.